src/reference/changes.rst

```diff
@@ -172,6 +191,42 @@ For more information, see the reference for the
 :cylc:conf:`[runtime][<namespace>]completion` configuration.
 
 
+Workflow State Triggers & Commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Workflow state xtriggers and command now take Cylc universal IDs instead of
+separate arguments:
+
+For example, you can (and should) now write:
+
+.. code-block:: diff
+
+   # On the command line
+   - cylc workflow-state my-workflow --point 20240101 --task mytask --message "succeeded"
+   + cylc workflow-state my-workflow//20240101/mytask:succeeded
+
+   # In the flow.cylc file
+   - my_xtrigger = workflow_state(
+   -     workflow="my-workflow",
+   -     task="mytask",
+   -     point="20240101",
+   -     message="succeeded"
+   - )
+   + my_xtrigger = workflow_state('my-workflow//20240101/mytask:succeeded')
+
+.. important::
+
+   The new workflow state triggers use the trigger rather than the message
+   where :cylc:conf:`[runtime][task][outputs]trigger=message`.
+   
+   The trigger and message are the same for the most common use cases (`succeeded` and `started`)
+   but may differ for other outputs, namely :term:`custom outputs <custom output>`.
```

Comment:
sry

```suggestion
   The new workflow state triggers use the trigger rather than the message
   where :cylc:conf:`[runtime][<namespace>][outputs]trigger=message`.
   
   The trigger and message are the same for the most common use cases (``succeeded`` and ``started``)
   but may differ for other outputs, namely :term:`custom outputs <custom output>`.
```

Reply:
No Worries